autogen/versions.tf

```diff
@@ -15,9 +15,15 @@
  */
 
 terraform {
-  required_version = ">=0.12.6, <0.14"
+  required_version = ">=0.13.0"
   required_providers {
-    google      = ">= 2.7, <4.0"
-    google-beta = ">= 2.7, <4.0"
+    google      = ">= 3.43, <4.0"
+    google-beta = ">= 3.43, <4.0"
+  }
+  provider_meta "google" {
+    module_name = "blueprints/terraform/terraform-google-vm/v6.0.0"
```

Comment:
We should inject the submodule name for submodules. (ex. `blueprints/terraform/terraform-google-vm-mig/v6.0.0`).

Reply:
done